packages/internal/src/__tests__/build_api.test.ts

```diff
@@ -148,3 +336,97 @@ test('jest mock statements also handle', () => {
   // Step 3: check that output has correct jest.mock path
   expect(outputForJest).toContain('jest.mock("../../lib/dog"')
 })
+
+test('core-js polyfill list', () => {
+  const { list } = compat({
+    targets: { node: TARGETS_NODE },
+    version: 3,
+  })
+
+  expect(list).toMatchInlineSnapshot(`
+    Array [
+      "es.math.hypot",
+      "es.typed-array.set",
+      "esnext.aggregate-error",
+      "esnext.array.last-index",
+      "esnext.array.last-item",
+      "esnext.composite-key",
+      "esnext.composite-symbol",
+      "esnext.map.delete-all",
+      "esnext.map.every",
+      "esnext.map.filter",
+      "esnext.map.find",
+      "esnext.map.find-key",
+      "esnext.map.from",
+      "esnext.map.group-by",
+      "esnext.map.includes",
+      "esnext.map.key-by",
+      "esnext.map.key-of",
+      "esnext.map.map-keys",
+      "esnext.map.map-values",
+      "esnext.map.merge",
+      "esnext.map.of",
+      "esnext.map.reduce",
+      "esnext.map.some",
+      "esnext.map.update",
+      "esnext.math.clamp",
+      "esnext.math.deg-per-rad",
+      "esnext.math.degrees",
+      "esnext.math.fscale",
+      "esnext.math.iaddh",
+      "esnext.math.imulh",
+      "esnext.math.isubh",
+      "esnext.math.rad-per-deg",
+      "esnext.math.radians",
+      "esnext.math.scale",
+      "esnext.math.seeded-prng",
+      "esnext.math.signbit",
+      "esnext.math.umulh",
+      "esnext.number.from-string",
+      "esnext.observable",
+      "esnext.promise.any",
```

Comment:
`esnext` means it's a proposal. You can tell this list is outdated by going to the core-js repo and searching for `esnext.promise.any`—it's `es.promise.any` now. See https://github.com/zloirock/core-js#ecmascript-promise